### DISCUSSION TITLE
I want to click on the plugin's icon to open the sidebar, how should I operate?

### DISCUSSION THREAD

**Discussion by elonehoo** [wxt-dev/wxt#921] [General]
> I want to click on the plugin's icon to open the sidebar, how should I operate?

**Comment by 1natsu172** [comment]
@elonehoo You could combine action and sidepanel API? I have never used the sidepanel API myself, but maybe.

- https://developer.chrome.com/docs/extensions/reference/api/action
- https://developer.chrome.com/docs/extensions/reference/api/sidePanel

**Comment by aiktb** [comment]
same https://github.com/wxt-dev/wxt/issues/570
